### PR TITLE
Update exodus-intro.md

### PR DESCRIPTION
**Pull request by natansil**

### CODE REVIEW

File: docs/exodus-intro.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-# Exodus - Maven to Bazel Migration
+# Exodus - Maven to Bazel Migration 
 
 Thinking about migrating from Maven or similar to another build system? This is our story at Wix. For some more background info, take a look at our [blog articles on Medium](https://medium.com/wix-engineering/migrating-to-bazel-from-maven-or-gradle-5-crucial-questions-you-should-ask-yourself-f23ac6bca070).
 
-## Maven had been good to us...for a time
+## Maven had been good to us...for a time 
 
 Like many developers, you have probably been using Apache Maven to build your projects. We got used to managing those POM (project object model) files and waiting for our projects to build, hoping we had no errors and those pesky dependencies would all work out.
```
Comment (natansil) [inline docs/exodus-intro.md:7]: java developers

Comment (natansil) [inline docs/exodus-intro.md:7]: managing POMs is acutally easier than managing bazel BUILD files...

File: docs/exodus-intro.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+# Exodus - Maven to Bazel Migration 
 
 Thinking about migrating from Maven or similar to another build system? This is our story at Wix. For some more background info, take a look at our [blog articles on Medium](https://medium.com/wix-engineering/migrating-to-bazel-from-maven-or-gradle-5-crucial-questions-you-should-ask-yourself-f23ac6bca070).
 
-## Maven had been good to us...for a time
+## Maven had been good to us...for a time 
 
 Like many developers, you have probably been using Apache Maven to build your projects. We got used to managing those POM (project object model) files and waiting for our projects to build, hoping we had no errors and those pesky dependencies would all work out. 
 
 Maven was good to us when our repositories were a certain size and our projects didn't take hours to build.
```
Comment (natansil) [inline docs/exodus-intro.md:9]: size and dependency complexity.
build took hours to finish in CI (on infra code change)

File: docs/exodus-intro.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 At Wix we realized we needed a new build manager.
 
-## Why Bazel
+## Why Bazel 
 
 After evaluating several solutions, **Bazel** was chosen as the best alternative build tool.
 
-### Faster builds and tests
+### Faster builds and tests 
 The most significant advantage of Bazel is the speed of the build process. What used to take us xx now takes us yy.
```
Comment (natansil) [inline docs/exodus-intro.md:18]: xx hours
yy a few minutes

File: docs/exodus-intro.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 
 How does Bazel perform so fast?
```
Comment (natansil) [inline docs/exodus-intro.md:20]: maybe provide links to bazel docs?

File: docs/exodus-intro.md [hunk 5]
```diff
@@ -23,16 +23,16 @@ It uses:
 * Parallelism - can run compilations and tests in parallel.
 * Hermeticity - builds can access only what is declared and pre-fetched, so they run 'hermetically' without unnecesary noise from additional files. 
 
-### Multi-language environment
```
Comment (natansil) [inline docs/exodus-intro.md:26]: currently we migrated jvm languages, but we are thinking of migrating node.js and maybe even react as well

File: docs/exodus-intro.md [hunk 6]
```diff
@@ -23,16 +23,16 @@ It uses:
...
+### Multi-language environment 
 Here at Wix, we support different software languages and it's a huge advantage to be able to build multiple languages simultaneously. 
 
-### Scalability
+### Scalability 
 Bazel can handle multiple repositories or any size mono-repo. We are able to scale our codebase and our continuous integration system.
```
Comment (natansil) [inline docs/exodus-intro.md:30]: due to build workers farm (read about bazel remote execution) we can scale out build actions to hunderds of workers simultaneously. so we have room to grow our code. >10MLOC in jvm languages

File: docs/exodus-intro.md [hunk 7]
```diff
@@ -23,16 +23,16 @@ It uses:
...
 
-### Extend languages and platforms
+### Extend languages and platforms 
 Bazel enables us to extend to more languages and platforms using Bazel's familiar extension language. Because there is a growing community of Bazel uses, we can share and re-use language rules written by other teams.
```
Comment (natansil) [inline docs/exodus-intro.md:33]: we maintain and contrbute to rules_scala  https://github.com/bazelbuild/rules_scala
there is a big thriing comunity of rules contributers

**Review by natansil — COMMENTED** [review]
@AnnieLR generally looks good!
I have a few corrections and enhancements...

File: docs/exodus-intro.md (hunk 1 above)
Comment (AnnieLR) [reply docs/exodus-intro.md:7]: Add "Java or Scala" developers

**Comment by natansil** [comment]
Hi @AnnieLR are you going to incorporate the changes, or do you want me to do it.

**Pull request closed without merging**